### PR TITLE
[DO NOT MERGE] Rename "Macedonia" to "North Macedonia"

### PR DESCRIPTION
**Pull request by deborahchua**
Trello card: https://trello.com/c/ySMq2bcm/786-slug-and-tag-changes-country-name-change-macedonia-to-north-macedonia

### CODE REVIEW

File: test/data/marriage-abroad-questions-and-responses.yml [hunk 1]
```diff
@@ -1,99 +1,163 @@
 ---
 :country_of_ceremony?:
+- afghanistan
```
Comment (deborahchua) [inline test/data/marriage-abroad-questions-and-responses.yml]: I'm not sure if these countries should have been added in but was the result of running the regression tests.

File: test/artefacts/marriage-abroad/madagascar/ceremony_country/partner_british/opposite_sex.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Marriage in Madagascar
```
Comment (edwardkerry) [inline test/artefacts/marriage-abroad/madagascar/ceremony_country/partner_british/opposite_sex.txt]: Are these files meant to be in here? Madagascar, and Malawi also further down.

Comment (AlanGabbianelli) [reply test/artefacts/marriage-abroad/madagascar/ceremony_country/partner_british/opposite_sex.txt]: This is the result of running `bin/prep_for_pull_request.sh marriage-abroad`. It was probably missed in a previous PR.

**Comment by deborahchua** [comment]
Closing this PR as reworked in https://github.com/alphagov/smart-answers/pull/3899

**Pull request closed without merging**